build/ci/jenkins_pipeline_build

```diff
@@ -15,9 +15,9 @@ pipeline {
                 sh './build/ci/run_unittests.sh'
             }
         }
-        stage ('Run OLM verifications') {
+        stage ('Run olm validations') {
```

Comment:
please keep the OLM in upper case for this stage title

Reply:
Done